File: java/Translator.java
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.net.URLEncoder;
import java.nio.charset.StandardCharsets;
import java.util.Scanner;

public class Translator {
    private static final String SERVER = "http://localhost:8888";

    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter error message: ");
        String input = sc.nextLine();
        sc.close();

        String url = SERVER + "/translate?q=" + URLEncoder.encode(input, StandardCharsets.UTF_8);
        HttpClient client = HttpClient.newHttpClient();
        HttpRequest req = HttpRequest.newBuilder().uri(URI.create(url)).build();
        HttpResponse<String> resp = client.send(req, HttpResponse.BodyHandlers.ofString());

        System.out.println(resp.body());
    }
}

File: csharp/Translator.cs
using System;
using System.Net.Http;
using System.Web;

class Translator {
    static async Task Main(string[] args) {
        Console.Write("Enter error message: ");
        string input = Console.ReadLine() ?? "";

        using HttpClient client = new();
        string url = "http://localhost:8888/translate?q=" + Uri.EscapeDataString(input);
        string result = await client.GetStringAsync(url);

        Console.WriteLine(result);
    }
}